
---
float32[] x_coords_chair
float32[] y_coords_chair
---